### DISCUSSION TITLE
PiPedal v1.1.31 arm64

### DISCUSSION THREAD

**Discussion by rerdavies** [rerdavies/pipedal#132] [Releases]
Very minor fix.

- Add third "Use System Preferences" option for dark mode when using the Android app. (Dark mode follows Android system preferences by default).

<hr /><em>This discussion was created from the release <a href='https://github.com/rerdavies/pipedal/releases/tag/'>PiPedal v1.1.31 arm64</a>.</em>

**Comment by Peetem2023** [comment]
Thanks for your work on this project!
e the error
So, I downloaded and attempted to install.  I was given several errors.  Being a Pi noob, not sure how to proceed.  Any suggestions?  Here are the errors:

ilding dependency tree... Done
Reading state information... Done
Note, selecting 'pipedal:arm64' instead of './pipedal_1.1.31_arm64.deb'
Some packages could not be installed. This may mean that you have
requested an impossible situation or if you are using the unstable
distribution that some required packages have not yet been created
or been moved out of Incoming.
The following information may help to resolve the situation:

The following packages have unmet dependencies:
 pipedal:arm64 : Depends: lv2-dev:arm64 but it is not installable
                 Depends: dnsmasq:arm64 but it is not installable
                 Depends: authbind:arm64 but it is not installable
                 Depends: libasound2:arm64 (>= 1.0.29) but it is not installable
                 Depends: libavahi-client3:arm64 (>= 0.6.16) but it is not installable
                 Depends: libavahi-common3:arm64 (>= 0.6.16) but it is not installable
                 Depends: libboost-iostreams1.74.0:arm64 (>= 1.74.0) but it is not installable
                 Depends: libc6:arm64 (>= 2.30) but it is not installable
                 Depends: libflac++6v5:arm64 (>= 1.3.1) but it is not installable
                 Depends: libgcc-s1:arm64 (>= 3.0) but it is not installable
                 Depends: liblilv-0-0:arm64 (>= 0.22.0~dfsg0) but it is not installable
                 Depends: libnl-3-200:arm64 (>= 3.2.7) but it is not installable
                 Depends: libnl-genl-3-200:arm64 (>= 3.2.7) but it is not installable

**Comment by Peetem2023** [comment]
Figured it out!
